.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": [
+      ["env", {
+        "targets": {
+          "node": "6.5"
```

Comment:
Why node: 6.5?
In the requirements to the HW was asked to add support of es6. The command `npm install --save-dev babel-preset-es2016` is deprecated in the new babel version so I was not able to install it. 
Instead, I used `npm install babel-preset-env --save-dev` but it adds support not only of es6, but also babel-preset-es2015, and babel-preset-es2017 together. 
To use only es6 I found the fix in the article - https://github.com/babel/babel-preset-env/issues/316